runestone/poll/js/poll.js

```diff
@@ -59,7 +60,7 @@ export default class Poll extends RunestoneBase {
         this.containerDiv = document.createElement("div");
         this.pollForm = document.createElement("form");
         this.resultsDiv = document.createElement("div");
-        this.containerDiv.id = this.divid + "_container";
+        this.containerDiv.id = this.divid;
```

Comment:
I think this is how it should have been all along.

Reply:
I agree! My goal was to make only the changes necessary / reasonable to the components (provide a single, unique ID, etc.).